### ISSUE TITLE
Missing lang attribute ruins Accessibility score

### ISSUE THREAD

**Issue by mihkelal** [mihkelal/todonaut#56]
If a page doesn't specify a lang attribute, a screen reader assumes that the page is in the default language that the user chose when setting up the screen reader. If the page isn't actually in the default language, then the screen reader might not announce the page's text correctly.

<img width="969" alt="Screenshot 2023-03-10 at 14 05 25" src="https://user-images.githubusercontent.com/12763881/224311793-39791ea1-1c38-4c27-8be4-1da61e5c619d.png">
